Standard/BisCustomAttributes.ecschema.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- ==================================================================================
+|  * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+|  * See LICENSE.md in the project root for license terms and full copyright notice.
+======================================================================================= -->
+<ECSchema schemaName="BisCustomAttributes" alias="bisCA" version="01.00.00" description="Custom attributes to indicate BIS concepts." displayLabel="BIS Custom Attributes" xmlns="http://www.bentley.com/schemas/Bentley.ECXML.3.2">
+    <ECEnumeration typeName="SchemaGradeValue" backingTypeName="string" isStrict="true" description="Used with SchemaType to declare the schema author's intended grade for the schema.">
```

Comment:
What is 'SchemaType'?

Overall the description seems a bit odd.  I think the description should focus on what this enumeration defines, for example: Grades that define how well a schema follows BIS modeling principals.  I don't love that description either